src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/kafka/KafkaOppfolgingstilfellePerson.kt

```diff
@@ -25,7 +25,7 @@ data class KafkaOppfolgingstilfelle(
 )
 
 fun KafkaOppfolgingstilfellePerson.toLatestOppfolgingstilfelle(): Oppfolgingstilfelle? =
-    oppfolgingstilfelleList.maxByOrNull {
+    this.oppfolgingstilfelleList.filter { it.start.isBefore(LocalDate.now()) }.maxByOrNull {
```

Comment:
Det har kanskje ikke noe å si i praksis, men kunne det vært en idé å tatt med alle tilfeller som starter i dag (selv om det evt. er noen timer fram i tid)?

Jeg tror man kan droppe `this` 👍

Reply:
Jeg funderte litt på det samme, men har ikke kommet på noe tilfelle der det spiller noen rolle?

Reply:
Vet heller ikke om det vil ha noe å si i praksis - men virker jo for så vidt riktig å inkludere de som starter i dag også.

Reply:
Ja, vi kan godt endre til it.start.isBefore(LocalDate.now().plusDays(1))

(evnt lage en extension-funksjon i LocalDate.tomorrow())

Reply:
Fikser i isoppfolgingstilfelle og isdialogmotekandidat, men viser seg at det ikke går an å lage statiske extension functions, så da blir det bare en fun tomorrow() =..

Reply:
👍 Jepp, fikset her nå.